src/views/onboarding/components/githubExamples/index.tsx

```diff
@@ -0,0 +1,9 @@
+import { NotFound } from '../notFound';
+
+export const GithubExamples = () => {
+  return (
+    <div>
```

Comment:
we don't need this extra div here ... and we don't need GithubExamples component at all if we use it only to add this extra div here